src/tests/java/htsjdk/samtools/CRAMEdgeCasesTest.java

```diff
@@ -33,6 +34,25 @@ public void testUnsorted() throws IOException {
         testRecords(records, records.iterator().next().getReadBases());
     }
 
+    // int test for CRAMException
+    // testing for a contig found in the reads but not in the reference
+    @Test
+    public void testContigNotFoundInRef() throws IOException {
+        final File CRAMFile = new File("testdata/htsjdk/samtools/cram/test.cram");
+        final File refFile = new File("testdata/htsjdk/samtools/cram/test.fa");
```

Comment:
Rename these test files so as to make it clear that they belong to this test -- eg., `testContigNotInRef.cram` and `testContigNotInRef.fa`